json/schema/fairtracks_track.schema.json

```diff
@@ -217,11 +230,12 @@
     },
     "required": [
         "local_id",
+        "assembly_id",
+        "assembly_name",
```

Comment:
Are ID and name not redundant?

Reply:
No, unfortunately. See https://github.com/fairtracks/fairtracks_standard/issues/16 for a long discussion. Basically it is there in order to provide the UCSC-style assembly_names instead of the official ones.

Reply:
We do not need to make 'assembly_name' required, though, but I thought that would help make people aware of the issue, as the default official assembly_name would be incorrect for most tracks out there..

Reply:
I'd agree that assembly_name is not required, as this file complements the actual TrackHub file which has all the display related info.

Reply:
@dzerbino So I don't follow your argument here. FAIRtracks is designed to work independently of the TrackHub format, for use in e.g. analysis, and knowing the exact genome assembly (with the correct chromosome names and lengths) is fundamentally important, and this in a human readable way. assembly_id="insdc.gca:GCF_000001405.26" leads to "https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.26" by identifiers.org [except that there is currently a bug], and (as described in the NCBI link) the assembly_name can then be either the canonical "GRCh38" or its UCSC synonym "hg38". This difference is crucial for downstream tools to have correct, which is the reason I suggested assembly_name to be required. The alternative is to make it not required, but autogenerated, and in most cases this will end up with the canonical "GRCh38". However, for the vast majority of tracks, the correct would be "hg38" due to the "chr"-style chromosome names in the track files. If assembly_name is required, one now at least needs to manually make the decision of not reading the field description (which should explain the issue). If we remove the requirement, then we should have an alternative way of making sure the tracks annotated with FAIRtracks will be correctly annotated, so that downstream analysis tools don't have to continue to make special rules for this issue.

Reply:
Ah, OK, so basically assembly_name give you the coordinate system... gotcha.